frontend/js/game.js

```diff
@@ -2,134 +2,116 @@
  * Created by JanJJ on 21.10.2016.
  */
 
-var game={};
-game.player={};
-
-game.config = {
-    player: "player",
-    playerSymbol: "&#916;",
-    startPosX: 1,
-    startPosY: 1
-};
-
+var game = {};
 game.player = {
-  name: "",
-  collectedCandys: 0,
-  keyCount: 0
+    name: "",
+    collectedCandies: 0,
+    keyCount: 0
 };
 
-game.startTime = 0;
-
-game.player.setPos = function (x, y) {
+game.player.movePlayerTo = function (x, y) {
     var locationId = x + maze.config.splitChar + y;
-    if(!maze.isWall(x,y)){                    //Prüft ob nicht auf Mauer gelaufen wird
-        console.log("isWall: " + maze.isWall(x,y));
+    if (!maze.isWall(x, y)) { //Prüft ob nicht auf Mauer gelaufen wird
         if (maze.isCandy(x, y)) {
-	        jQuery("#" + x+maze.config.splitChar+y).html(maze.config.chars[0]);
-	        jQuery("#" + x+maze.config.splitChar+y).removeClass("candy");  
-	        game.player.collectedCandys++;
-	    }
-	    game.player.setPlayer(x,y);
-        jQuery("#"+game.player.getPos()).removeAttr(game.config.player,"");     //Altes Attribut für Position löschen
-        jQuery("#"+locationId).attr(game.config.player,"test");  //Neues Attribut für Position setzen
+            var $xyPosElement = $("#" + x + maze.config.splitChar + y);
+            $xyPosElement.html(maze.config.chars[0]);
+            $xyPosElement.removeClass("candy");
+            game.player.collectedCandies++;
+        }
+        game.player.setPos(x, y);
+        $("#" + game.player.getPos()).removeAttr(game.config.selectorPlayerClass, ""); //Altes Attribut für Position löschen
+        $("#" + locationId).attr(game.config.selectorPlayerClass, "test"); //Neues Attribut für Position setzen
         if (maze.isOnExit(x, y)) {
-        	game.exit();
-        }               
-    }else{
-        console.log("Unerlaubter Zug!");
+            game.finishGameAndDisplayText();
+        }
     }
 };
 
+game.player.getPos = function () {
+    return $("span[player]").attr("id");
+};
+
+game.player.setPos = function (x, y) {
+
+    //Alte Position löschen UND setze Klasse game.config.selectorPlayerClass zurück für die Farbe
+    var $playerPos = $("#" + game.player.getPos());
+    $playerPos.html(maze.config.chars[0]);
+    $playerPos.removeClass(game.config.selectorPlayerClass);
+
+    //Neue Position setzen UND setze Klasse game.config.selectorPlayerClass für die Farbe
+    var $newPlayerPos = $("#" + x + maze.config.splitChar + y);
+    $newPlayerPos.html(game.config.playerSymbol);
+    $newPlayerPos.addClass(game.config.selectorPlayerClass);
+};
+
+game.startTime = 0;
+
+game.config = {
+    selectorPlayerClass: "player",
+    playerSymbol: "&#916;",
+    startPosX: 1,
+    startPosY: 1
+};
 
-game.keyevent = function () {
-	if (game.startTime == 0) {
-		jQuery(document).off("keydown");
-		return;
-	}
-    jQuery(document).on("keydown", jQuery(document), function (event) {
+game.initialiseKeyEvent = function () {
```

Comment:
http://askubuntu.com/questions/195073/synaptic-initialize-initialise-why-the-difference